aws/data_source_aws_workspaces_image.go

```diff
@@ -0,0 +1,69 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/workspaces"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceAwsWorkspacesImage() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsWorkspacesImageRead,
+
+		Schema: map[string]*schema.Schema{
+			"image_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"os": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"required_tenancy": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
We can remove the `state` attribute, since it is more of a run-time property, not a configuration property.

Reply:
But this is a data source, not a resource. Don't we need to get info about any possible attribute there?

Reply:
Yes, typically we'd like the data source to return as many attributes as possible. In this case, though, `state` is something that can change at any time, even potentially could change one second after Terraform is run. So it doesn't fit with what Terraform should be working with.